Artifacts/linux-secure-mariadb/linux-secure-mariadb.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+#
+# Script to secure MariaDB installation on Linux.
+# Based on mysql_secure_installation.sh found here https://github.com/twitter/mysql/blob/master/scripts/mysql_secure_installation.sh
+#
+# NOTE: Intended for use by the Azure DevTest Lab artifact system.
+#
+# Usage: 
+#
+# linux-secure-mariadb.sh [-p=password]

```

Comment:
Again, just pointing out this example does't match the `USAGE_STRING` defined just below.